.github/workflows/mvn-verify-check.yml

```diff
@@ -30,7 +30,45 @@ env:
     -Dmaven.scaladoc.skip
 
 jobs:
+  cache-dependencies:
+    runs-on: ubuntu-latest
+    outputs:
+      dailyCacheKey: ${{ steps.generateCacheKey.outputs.dailyCacheKey }}
+    steps:
+      - uses: actions/checkout@v3 # refs/pull/:prNumber/merge
+      - uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin'
+          java-version: 8
+      - name: populate-daily-cache
+        run: |
+          set -x
+          max_retry=3; delay=30; i=1
+          while true; do
+            mvn help:evaluate -pl dist -Dexpression=included_buildvers \
+            -DforceStdout -PnoSnapshots -q | tr -d ',' | \
+            xargs -n 1 bash -c 'mvn initialize -pl sql-plugin-api -am -Dbuildver=$1' _ && break || {
+            if [[ $i -le $max_retry ]]; then
+              echo "mvn command failed. Retry $i/$max_retry."; ((i++)); sleep $delay; ((delay=delay*2))
+            else
+              echo "mvn command failed. Exit 1"; exit 1
+            fi
+          }
+          done 
+      - name: Generate daily cache key
```

Comment:
move steps `Generate daily cache key` and `Cache local Maven repository` right above `populate-daily-cache` on L43